runtime/lua/vim/shared.lua

```diff
@@ -544,14 +544,9 @@ function vim.spairs(t)
   end
   table.sort(keys)
 
-  -- Return the iterator function.
-  -- TODO(justinmk): Return "iterator function, table {t}, and nil", like pairs()?
```

Comment:
To be clear, this comment is about `:help for-in`. But I think this comment should be removed, it's not relevant for pairs()-like iteration. Fixed in https://github.com/neovim/neovim/pull/24561